### ISSUE TITLE
OpenGL on Macbook?

### ISSUE THREAD

**Issue by slerman12** [deepmind/dm_control#207]
It's great to see that MuJoCo is open-sourced and the installation process is more streamlined! I'm testing if the Mac version would work on my M1 MacBook, and I keep getting:

```
    raise ImportError("Unable to load EGL library", *err.args)
ImportError: ('Unable to load EGL library', 'dlopen(EGL, 10): image not found', 'EGL', None)
```

I tried the following:
`pip install PyOpenGL-accelerate`
and
`conda install pyopengl`

Both installed successfully, but I get the same error.

**Comment by slerman12** [comment]
Same result from just running the export command, then running my python script. Do you mean, uninstall dm_control, run the export command, then reinstall dm_control and try again?

**Comment by matte-esse** [comment]
please update mujoco tu support m1 macs!

**Comment by saran-t** [comment]
Looks like the issue above isn't related to M1-ness at all, but rather is an issue with GLFW in Conda. I'm going to close this issue, and let's continue the discussion in #223 .

(Separately, MuJoCo 2.1.0 wouldn't have worked under Rosetta2 anyway because it uses AVX instructions. This is now resolved through the 2.1.1 release.)
